main/FatesInterfaceMod.F90

```diff
@@ -1800,11 +1723,8 @@ subroutine set_fates_ctrlparms(tag,ival,rval,cval)
                end if
 
             case default
-               if (fates_global_verbose()) then
-                   write(fates_log(), *) 'tag not recognized:',trim(tag)
-                   call endrun(msg=errMsg(sourcefile, __LINE__))
-               end if
-               ! end_run
+               write(fates_log(), *) 'fates NL tag not recognized:',trim(tag)
+               !! call endrun(msg=errMsg(sourcefile, __LINE__))
```

Comment:
We are currently passing in a tag that is not recognized!  I don't know where this came from, but 

https://github.com/ESCOMP/CTSM/blob/ctsm5.1.dev090/src/utils/clmfates_interfaceMod.F90#L323

I don't see this passed in E3SM:

https://github.com/E3SM-Project/E3SM/blob/master/components/elm/src/main/elmfates_interfaceMod.F90#L375-L379

Reply:
I'm guessing 'sf_anthro_suppression_def' was added at one point with the expectation it would be used, but the usage was never implemented.  Lets remove this argument pass in the next API change on ctsm

Reply:
@rgknox I am guessing this is a tag associated with anthropogenic fire suppression. You are correct that it is not yet in use. I think @glemieux or @slevisconsulting implemented this?

Reply:
Yup that was me :facepalm:.  As Jackie noted its part of https://github.com/ESCOMP/CTSM/pull/1536.  Since this was a ctsm-side only update to prep for future work I guess should have commented out those lines and the lines later on down the file to avoid this issue?

https://github.com/ESCOMP/CTSM/blob/449345ee9a534316cba0d6b4fd696b8d57e60428/src/utils/clmfates_interfaceMod.F90#L822-L829

and

https://github.com/ESCOMP/CTSM/blob/449345ee9a534316cba0d6b4fd696b8d57e60428/src/utils/clmfates_interfaceMod.F90#L846-L850